File: src/main/kotlin/mocking/emis/models/AppointmentSlot.kt
package mocking.emis.models

data class AppointmentSlot(
        var slotId: Int? = null,
        var startTime: String? = null,
        var endTime: String? = null,
        var slotTypeName: String? = null,
        var slotTypeStatus: SlotTypeStatus? = null
)

File: backendworker/NHSOnline.Backend.Worker/GpSystems/Suppliers/Emis/Models/AppointmentSlot.cs
namespace NHSOnline.Backend.Worker.GpSystems.Suppliers.Emis.Models
{
    public class AppointmentSlot
    {
        public int SlotId { get; set; }
        public string StartTime { get; set; }
        public string EndTime { get; set; }
        public string SlotTypeName { get; set; }
    }
}